doc/conf.py

```diff
@@ -255,6 +255,36 @@
     major, minor = match.groups()
     binder_branch = '{}.{}.X'.format(major, minor)
 
+
+class SubSectionTitleOrder:
+    """Sort by title of subsection.
+
+    Assumes README.txt exists for all subsections and uses the subsection with
+    dashes, '---', as the adornment.
+    """
+    def __init__(self, src_dir):
+        self.src_dir = src_dir
+        self.regex = re.compile(r"^([\w ]+)\n-", re.MULTILINE)
+
+    def __repr__(self):
+        return '<%s>' % (self.__class__.__name__,)
+
+    def __call__(self, directory):
+        src_path = os.path.normpath(os.path.join(self.src_dir, directory))
+        readme = os.path.join(src_path, "README.txt")
+
+        try:
+            with open(readme, 'r') as f:
+                content = f.read()
+        except FileNotFoundError:
+            return directory
+
+        title_match = self.regex.search(content)
+        if title_match is not None:
+            return title_match.group(1)
+        return directory
```

Comment:
that returns "../examples"?

Reply:
This will return `../examples/neighbors`, if the subsection title can not be found.